poc.js

```diff
@@ -1,212 +1,211 @@
 const MongoClient = require('mongodb').MongoClient;
 const elasticsearch = require('elasticsearch');
-const esClient = new elasticsearch.Client({ host: 'localhost:9200', apiVersion: '2.4' });
+const esClient = new elasticsearch.Client({host: 'localhost:9200', apiVersion: '2.4'});
 const fs = require('fs');
 const f = require('util').format;
-const BSON = require('bson')
+const BSON = require('bson');
 const bson = new BSON();
 let url = process.env.MONGO_RS || 'mongodb://localhost:27017';
 const dbName = 'vrs';
-const collectionName = 'voter';
 let resumeToken;
-const ARG_SKIP = Number(process.argv[2]) || 0;
-const ARG_LIMIT = Number(process.argv[3]) || 100000;
-const ARG_DUMP = process.argv[4] === '-d' ? true : false;
+
+//1st, 2nd, & 3rd command line args:
+const ARG_SKIP = Number(process.argv[2]) || 0; // Set to skip N documents ahead in your query
+let ARG_LIMIT = Number(process.argv[3]) || 100000; // the bulk insertion limit
+const ARG_DUMP = process.argv[4] === '-d'; // whether or not to dump all collections into elasticsearch
 const BULK_SIZE = 1000;
 let bulkOpsDone = 0;
 const testDoc = require('./testDoc');
 
+const esFunctions = {
+    'insert': insertDoc,
+    'update': updateDoc,
+    'replace': replaceDoc,
+    'delete': deleteDoc,
+    'invalidate': invalidate
+};
 
+// parse the resume token if it exists
 try {
-  const base64Buffer = fs.readFileSync('./resumeToken');
-  resumeToken = bson.deserialize(base64Buffer);
+    const base64Buffer = fs.readFileSync('./resumeToken');
```

Comment:
Are these tabs? We should be careful to use spaces consistently. I suggest running everything through `eslint`. We have an E1C-specific style configuration we use.